README.md

```diff
@@ -167,6 +168,15 @@ slog.Info("a user has logged in",
 )
 ```
 
+### Arguments with forbidden keys
+
+To ensure users don't use reserved words in key-value pairs, you may want to enforce rules to avoid misusages.
+The `forbidden-key` option causes `sloglint` to report usages of forbidden keys. These keys could be typically controlled by slog handlers and its usage should be preserved to avoid key duplication.
```

Comment:
Can we document how to set this option, e.g. "Keys to forbid can either be comma separated or passed as separate `forbidden-key` options."